package-lock.json

```diff
@@ -152,6 +162,18 @@
       "integrity": "sha1-wuep93IJTe6dNCAq6KzORoeHVYA=",
       "dev": true
     },
+    "mdn-data": {
```

Comment:
Will importing mdn-data from npm cause any problems, e.g. if you add a new type in the dataset will the tests start failing until a new release of the npm package is out?

Reply:
That `mdn-data` npm package is there because css-tree depends on it, but it's never used. The test loads the local data (e.g. `require('../css/properties')`).